visualization/app/codeCharta/ui/ribbonBar/ribbonBar.component.ts

```diff
@@ -6,8 +6,8 @@ import { DownloadService } from "../../core/download/download.service";
 export class RibbonBarController {
 
     private collapsingElements = $("code-map-component #codeMap, ribbon-bar-component #header, ribbon-bar-component .section-body, #toggle-ribbon-bar-fab");
-    private deltaMode = KindOfMap.Delta;
     private isExpanded: boolean = false;
+    deltaMode = KindOfMap.Delta;
```

Comment:
Why is this no longer private?

Reply:
good point. Now idea.. I set it back again